packages/renderer/src/constants/lang/LangKeys.ts

```diff
@@ -20,10 +20,18 @@ export enum LangKeys {
   Header = "app.header",
   ConnectingToNetwork = "app.connectingToNetwork",
   WelcomeToHaveno = "app.welcomeToHaveno",
+  Save = "app.save",
   AccountTitle = "account.title",
   AccountSidebarPaymentAccounts = "account.sidebar.paymentAccounts",
   AccountSidebarSecurity = "account.sidebar.security",
   AccountSidebarWallet = "account.sidebar.wallet",
   AccountSidebarBackup = "account.sidebar.backup",
   AccountSidebarNodeSettings = "account.sidebar.nodeSettings",
+  AccountSecurityTitle = "account.security.title",
+  AccountSecurityDesc = "account.security.desc",
+  AccountSecurityFieldPassword = "account.security.field.password",
+  AccountSecurityFieldRepeatPassword = "account.security.field.repeatPassword",
+  AccountSecurityFieldCurrentPassword = "account.security.field.currentPassword",
+  AccountSecurityFieldPasswordFormatMsg = "account.security.field.password.format.message",
+  AccountSecurityFieldRepeatPasswordMatchMsg = "account.security.field.repeatPassword.match.message",
```

Comment:
You're very thorough! 🤩

Reply:
Yeah, to avoid any conflicts in the future. :)